docs/manual/src/internals/foreign_calls.md

```diff
@@ -0,0 +1,84 @@
+# Rust -> Foreign calls
+
+How can Rust code make a call into the foreign language?
+The [lifting and lowering](./lifting_and_lowering.md) docs describe this at a high-level, this document will explore the details.
+
+This document only describes non-async calls.
+See the [Async FFI](./async-ffi.md) docs for a description of async calls.
+
+## VTables
+
+One major difference from a [Foreign -> Rust call](./rust_calls.md), is that the Rust code does not have access to a library with functions it can call.
+Because of this, the foreign code registers a "VTable" with the Rust code.
+This is a `repr(C)` struct where each method is a function pointer corresponding to a method in a callback interface.
```

Comment:
s/method/entry/ or something?  (ie, I don't really think of a struct having a "method", unless it is a function pointer, which is the point you are making?)